freertos-cargo-build/src/lib.rs

```diff
@@ -61,6 +62,7 @@ impl Builder {
         let freertos_path = env::var(ENV_KEY_FREERTOS_SRC).unwrap_or_default();
         let freertos_config_path = env::var(ENV_KEY_FREERTOS_CONFIG).unwrap_or_default();
         let freertos_shim = env::var(ENV_KEY_FREERTOS_SHIM).unwrap_or_default();
+        let freertos_port_base = format!("{}{}", freertos_path, "portable");
```

Comment:
Would prefer to use something like `Path::join` here to ensure the separator is correct in all cases.